### PR TITLE
Feature/main menu

### PR DESCRIPTION
**Pull request by michalpokojski**

### CODE REVIEW

File: src/SecuredRoute.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+
+
```
Comment (bachbach) [inline src/SecuredRoute.js]: remove line

File: src/components/MainMenu.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Tabs, Tab } from 'material-ui/Tabs';
+import { withRouter } from 'react-router-dom'
+
+class MainMenu extends Component {
+
+  render() {
+    return (
+      <Tabs>
+        <Tab onActive={() => this.props.history.push('/dashboard')} label="Dashboard"/>
```
Comment (bachbach) [inline src/components/MainMenu.js]: Please avoid functions definition in render. Use arrow function in class instead:
```
<Tab onActive={this.redirect('/dashboard')} label="Dashboard"/>
```
And in class:
```
redirect = (path) => () => this.props.history.push(path)
```

File: src/components/MainMenu.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        <Tab onActive={() => this.props.history.push('/dashboard/posts')} label="Posts"/>
+        <Tab onActive={() => this.props.history.push('/dashboard/users')} label="Users"/>
+        <Tab onActive={() => this.props.logOutUser()} label="Logout"/>
```
Comment (bachbach) [inline src/components/MainMenu.js]: the same here

File: src/index.css [hunk 4]
```diff
@@ -1,5 +1,11 @@
-body {
-    margin: 0;
-    padding: 0;
-    font-family: sans-serif;
-}
+/*body {*/
```
Comment (bachbach) [inline src/index.css]: is this comment needed?

File: src/SecuredRoute.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+export default function (ComposedComponent) {
+  class Authentication extends Component {
```
Comment (bachbach) [inline src/SecuredRoute.js]: Please, keep classname ( if one in file ) same as file name

File: src/App.js [hunk 6]
```diff
@@ -5,22 +5,27 @@ import Router from './router'
...
-          <div>
-            <AuthRoute path='/'/>
-            <Route path='/login' component={LogIn}/>
-            <Route path='/dashboard' component={AppView}/>
-          </div>
+          <MuiThemeProvider>
+            <div>
+              <AuthRoute path='/'/>
+              <Route path='/login' component={LogIn}/>
+              <Route path='/dashboard' component={SecuredRoute(AppView)}/>
```
Comment (bachbach) [inline src/App.js]: Isn't `AuthRoute` do the authentication job?

Comment (michalpokojski) [reply src/App.js]: ```AuthRoute``` doesn't handle back button as specified in user stories. And yet ```SecuredRoute``` doesnt handle redirecting to /dashboard if there is ```user``` logged in and badly want to log in again so typing ```/login``` redirected him to login page

**Pull request merged**